package.json

```
@@ -150,6 +151,7 @@
 		"tar": "^7.4.0",
 		"unzipper": "0.10.11",
 		"url-loader": "^4.1.1",
+		"winreg": "^1.2.4",
```

```suggestion
		"winreg": "1.2.4",
```
I don't have a strong opinion about this, but regedit seems ever so slightly more well-maintained than winreg. The gotcha you linked to with Electron apps doesn't look too bad either, IMO. The fact that we can't update to the latest version of winreg and that 1.2.4 is 8 years old is a bit of a red flag, IMO.

If we do stick with winreg, we should enforce the "don't upgrade to 1.2.5" constraint in `package.json` according to my suggestion.

---

>I don't have a strong opinion about this, but regedit seems ever so slightly more well-maintained than winreg. The gotcha you linked to with Electron apps doesn't look too bad either, IMO. The fact that we can't update to the latest version of winreg and that 1.2.4 is 8 years old is a bit of a red flag, IMO.

The thing that disappointed me is - it supports async, but I got issues with it as soon as I added that fix for Electron. But yeah, maybe it makes sense to try to spend more time with it to make it working well.
What I liked with winreg - it was only one package which started working w/o extra effort and make me feeling reliable.

I think that the best way is to go with winreg ATM, since I tested it well and everything is ready to move forward, And later, if we encounter some issues with it, then we can reconsider to take regedit or maybe even something else. But ARTM winreg does it's job well.

---

>If we do stick with winreg, we should enforce the "don't upgrade to 1.2.5" constraint in package.json according to my suggestion.

Hm, I did install the exect version, weird that at some point it was reverted... got catch, removed `^`👍